### DISCUSSION TITLE
Release v1.7.2 / 5.62.2

### DISCUSSION THREAD

**Discussion by DavidXanatos** [sandboxie-plus/Sandboxie#2668] [Announcements]
![0 5 5](https://user-images.githubusercontent.com/3890945/105014988-575cb680-5a41-11eb-90d5-d942d28184b2.png)

This build enables shadow stack for the SbieDll.dll improving on security on systems with gen11 Intel and Ryzen 5000 AMD CPUs.
This build restructures the Box options a bit and adds a whole new page allowing to control the file migration behavior as well as adding a couple SBIE messages to indicate file migration operations which may cause compatibility issues.
This build also provides a new Box creation wizard and improves on the box picker dialog.
It is now possible to create a new sandbox from the run sandboxes dialog and if need be set it as temporary, so that it gets removed after the program terminates. The box picker dialog was improved and contains now a search function to be accessed with Ctrl+F. 
The boxes now created use by default the Delete V2 scheme in case you experience any issues with new boxes, this can be changed back in the box options to the old Delete V1, no old boxes will be migrated to Delete V2.
To improve on security the SandMan UI now indicates which processes have an administrative and which a system token.
Furthermore with this build it is possible to define file checker triggers to for example invoke a script to run a virus scan on a file before it is allowed to be recovered.

Last but not least this build fixes a lot of issues, please see [Changelog](https://github.com/sandboxie-plus/Sandboxie/blob/master/CHANGELOG.md#172--5622---2023-02-05) for more details.

#### You can support the project through donations, any help will be greatly appreciated.

[<img src="https://www.paypalobjects.com/en_US/AT/i/btn/btn_donateCC_LG.gif">](https://xanasoft.com/go.php?to=donate)

#### If you have issues with an update installation, just uninstall the previous version keeping the sandboxie.ini and reinstall the new build.

<hr /><em>This discussion was created from the release <a href='https://github.com/sandboxie-plus/Sandboxie/releases/tag/v1.7.2'>Release v1.7.2 / 5.62.2</a>.</em>

**Comment by BilleBarrett** [comment]
Can I get some direction as to why I'm getting this error
Thank you
chrome.exe (11600): SBIE2308 Could not create object directory:  [44 / C0000024]

**Reply by isaak654** [reply]
Did it work in the [1.6.7](https://github.com/sandboxie-plus/Sandboxie/releases/tag/v1.6.7) version?

**Comment by BilleBarrett** [comment]
I would think so, this is one of Dallas clients that I only check in on once in awhile.

 



My Staff 
"Your Windows System and Data are only as good as your last backup"
(336) 988-0891    ***@***.***> ***@***.*** 
To subscribe to my FYI emails send an email to
 ***@***.***> ***@***.***

**Reply by isaak654** [reply]
Just to be sure, please downgrade to the versions below and report which ones are working and which are not:
https://github.com/sandboxie-plus/Sandboxie/releases/tag/v1.7.1
https://github.com/sandboxie-plus/Sandboxie/releases/tag/v1.7.0
https://github.com/sandboxie-plus/Sandboxie/releases/tag/v1.6.7

**Comment by BilleBarrett** [comment]
Ok, but I have all these versions

**Comment by ghost** [comment]
https://i.imgur.com/TWaAkVX.png

**Comment by droped** [comment]
Why has 1.7.2 been replaced by 1.8.0? And why no 1.8.0 binaries?

**Reply by God-damnit-all** [reply]
Looks like 1.8.0 got pulled.

**Reply by hg421** [reply]
It's fixed now

**Comment by BilleBarrett** [comment]
Not sure what you mean, I just clicked on Edge then on YouTube and picked a video 
no delay

 

Remote PC Support
"Your Windows System and Data are only as good as your last backup"
(336) 988-0891    ***@***.***> ***@***.*** 
To subscribe to my FYI emails send an email to
 ***@***.***> ***@***.***

**Comment by BilleBarrett** [comment]
I tried mine with the latest Edge, no problem here

**Comment by BilleBarrett** [comment]
I just updated to the latest Nvidia drivers on my system and I have had no problems, also updated my wife computer with any problems

Sandboxie-Plus  1.8

 

Remote PC Support
"Your Windows System and Data are only as good as your last backup"
(336) 988-0891    ***@***.***> ***@***.*** 
To subscribe to my FYI emails send an email to
 ***@***.***> ***@***.***

**Comment by abet00** [comment]
Greetings, Guys.

Im having issue on my sandboxie classic using latest version on MS EDGE its not launching, any quickfix?

Version: Microsoft Edge Version 111.0.1661.44 (Official build) (64-bit)
